x-pack/plugins/fleet/public/components/agent_enrollment_flyout/types.ts

```diff
@@ -27,6 +35,8 @@ export interface BaseProps {
 
   isK8s?: K8sMode;
 
+  isCSP?: CSPObject;
```

Comment:
It will be hard for someone to guess what `isCSP` means without much context; what about changing it to `cloudSecurityIntegration`?

Also, I don't think we must follow the pattern for k8s, especially because we are not using `IS_LOADING` and `IS_NOT_CSP`, so it doesn't seem relevant, if allowed, I would like to suggest the following changes to make it simpler:


```ts
export type CloudSecurityIntegrationType = 'kspm' | 'cnvm' | 'cspm';

export interface CloudSecurityIntegration {
	integrationType: CloudSecurityIntegrationType | undefined;
  cloudFormationUrl: string | undefined;
}


export interface BaseProps {
//...
cloudSecurityIntegration?: CloudSecurityIntegration;
}
```